source/TestAdapter/Discover.cs

```diff
@@ -120,38 +120,40 @@ public static List<TestCase> FindTestCases(string source)
             Type[] allTypes = test.GetTypes();
             foreach (var type in allTypes)
             {
-                if (type.IsClass)
+                if (!type.IsClass)
```

Comment:
I get it that this is more a "code style preference" but I always find it hard in my mind to follow code when making negative comparisons... 😅

Reply:
Is it better now?